### PR TITLE
[packaging_uom] Add in README usage section

### PR DESCRIPTION
**Pull request by rafaelbn**
Add the option to activate to be able to manage packaging in Odoo

cc @rousseldenis

### CODE REVIEW

**Comment by rafaelbn** [comment]
packaging_uom

File: packaging_uom/README.rst [hunk 1]
```diff
@@ -26,6 +26,10 @@ To configure this module, you need to:
 Usage
 =====
 
+* To use this module Go to Inventory -> Configuration -> Settings and activate
+ in the section Products -> Packaging Methods  -> the option "Manage available
```
Comment (rousseldenis) [inline packaging_uom/README.rst:30]: Remove one space before '-> the option'

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor comment. Thanks for completion !

Could you fix pylint too ?

**Comment by rousseldenis** [comment]
@rafaelbn

File: packaging_uom/README.rst (hunk 1 above)
Comment (rafaelbn) [inline packaging_uom/README.rst:30]:
```suggestion
 in the section Products -> Packaging Methods -> the option "Manage available
```

**Comment by rafaelbn** [comment]
I opened https://github.com/OCA/stock-logistics-warehouse/pull/684

**Pull request closed without merging**